setup.py

```diff
@@ -60,7 +60,7 @@
         "datasets",
         "accelerate>=0.20.3,!=1.1.0",
         "pynvml==11.5.3",
-        "compressed-tensors"
+        "compressed-tensors==0.9.0"
```

Comment:
Maybe >=?

Reply:
yeah we should do this to pickup the nightly
updating